lib/train/extras/command_wrapper.rb

```diff
@@ -57,22 +57,23 @@ def verify
       return nil if res.exit_status == 0
 
       rawerr = res.stdout + " " + res.stderr
-
-      {
-        "Sorry, try again" => ["Wrong sudo password.", :bad_sudo_password],
-        "sudo: no tty present and no askpass program specified" =>
-          ["Sudo requires a password, please configure it.", :sudo_password_required],
-        "sudo: command not found" =>
+      msg, reason =
+        case rawerr
+        when /Sorry, try again/
+          ["Wrong sudo password.", :bad_sudo_password]
+        when /sudo: no tty present and no askpass program specified/
+          ["Sudo requires a password, please configure it.", :sudo_password_required]
+        when /sudo: command not found/
           ["Can't find sudo command. Please either install and "\
-            "configure it on the target or deactivate sudo.", :sudo_command_not_found],
-        "sudo: sorry, you must have a tty to run sudo" =>
+            "configure it on the target or deactivate sudo.", :sudo_command_not_found]
+        when /sudo: sorry, you must have a tty to run sudo/
           ["Sudo requires a TTY. Please see the README on how to configure "\
-            "sudo to allow for non-interactive usage.", :sudo_no_tty],
-      }.each do |sudo, human|
-        rawerr = human if rawerr.include? sudo
-      end
```

Comment:
Hey @zenspider, 

Here the only reason for switching to the `case` here is if an exception would occur I would rather say:
```
I am the one instead of I am one of them
```

Reply:
I'm not sure I understand that. Assuming my interpretation is correct, then you're doing "I am the one" via `raise`, not via `case` vs `{...}.each`.